src/controllers/authController.js

```diff
@@ -0,0 +1,30 @@
+import models from "../database/models/index.js";
+const {Users} = models;
+import helper from "../utils/jwtHelper.js";
+const { comparePassword } = require("../utils");
+
+export default class UserController{
+  static async login(req, res, next){
+        try{
+        const { email, password} = req.body;
+        const user = await Users.findOne({where:{email}});
+        if(!user){
+        return res.status(400).send({error: "Email not found"}); 
```

Comment:
```suggestion
        return res.status(400).json({error: "Email not found"}); 
```
send means turns in value into tex/html while json treats it as an object